r/R/examples/stat_large.R

```diff
@@ -0,0 +1,36 @@
+#source("../core.R")
+#source("../ui.R")
+library(h2owave)
+library(tokenizers)
+base_url <- "https://programminghistorian.org/assets/basic-text-processing-in-r"
+url <- sprintf("%s/sotu_text/236.txt", base_url)
+text <- paste(readLines(url), collapse = "\n")
+sentences <- tokenize_sentences(text)
+captions <- sentences[[1]][sample(seq(1,length(sentences[[1]])),1)]
+
+card_title <- sample(row.names(mtcars),1)
+card_value <- as.character(sample(mtcars[['mpg']],1))
+card_pc <- as.integer(runif(1,1,100))
+
+test_page <- page("/page_demo")
+test_page <- page.add(test_page,"/page_demo","test",ui_large_stat_card(box="1 1 2 2"
```

Comment:
We need to provide:
1. A canonical way to grab a page, given a URL.
2. A canonical way to add a card to a page.

In the above lines, having `page.add()` return a `test_page` is confusing to me.

In Python, `page.add()` is the mechanism to add a card to a page, and returns a reference to the card. In case the reference to the card is not required, the Python API provides a `page['card_name'] = ...` mechanism as well.

Reply:
The rdriver now is able to allocate a canonical object for a page by
`page <- Site("demo")`
similarly, cards can be added as follows
```
page$add.card(paste0("card_",column,"_",row),
                              ui_markdown_card(box=box,title=box,content=''))
```
you can drop the page as follows
`page$page.drop()`
or save the page (send to the server)
`page$page.save()`